src/main/scala/reactive/tweets/incoming/TweetActor.scala

```diff
@@ -0,0 +1,53 @@
+package reactive.tweets.incoming
+
+import akka.actor.{Props, Status, actorRef2Scala}
+import akka.persistence.{PersistentActor, SnapshotOffer}
+import reactive.tweets.domain.{Tweet, User}
+import reactive.tweets.incoming.TweetActor.{GetLastTen, LastTenResponse}
+import akka.persistence.SaveSnapshotSuccess
+
+object TweetActor {
+  def props(user: User): Props = Props(new TweetActor(user))
+
+  case class GetLastTen(user: User)
+  case class LastTenResponse(lastTen: List[Tweet])
+}
+
+class TweetActor(val user: User) extends PersistentActor {
+  override def persistenceId = user.name
+  var latestTweets = List[Tweet]()
+
+  override def receiveCommand = {
+    case tweet: Tweet =>
+      persist(tweet) { event =>
+        sender() ! Status.Success
```

Comment:
Applies to other places as well, btw, also one inside a string interpolation somewhere below :).